CHANGELOG.md

```diff
@@ -7,13 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+-   `Docker#run_with_args` method. This allows naming a container and assigning it to a specific docker network. The docker
```

Comment:
```suggestion
-   `Docker::run_with_args` method. This allows naming a container and assigning it to a specific docker network. The docker
```
Why using a hash symbol?

Reply:
Because I've wrote too much Java in my life.